<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>levelEditor</class>
 <widget class="QMainWindow" name="levelEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="addEntry">
        <property name="text">
         <string>Add Entry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyEntry">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Copy Entry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortEntries">
        <property name="text">
         <string>Sort Entries</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeEntry">
        <property name="text">
         <string>Delete Entry</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="entryView">
      <item>
       <widget class="QLabel" name="entryLabel">
        <property name="text">
         <string>Entries</string>
        </property>
        <property name="buddy">
         <cstring>entryTable</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="entryTable"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="pathButtons">
      <item>
       <widget class="QPushButton" name="addPath">
        <property name="text">
         <string>Add path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyPath">
        <property name="text">
         <string>Copy Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletePath">
        <property name="text">
         <string>Delete Path</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="pathView">
      <item>
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Entry Path</string>
        </property>
        <property name="buddy">
         <cstring>pathTable</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="pathTable"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
